php/php.code.analysis/src/org/netbeans/modules/php/analysis/commands/MessDetector.java

```diff
@@ -59,10 +60,12 @@
     private static final String REPORT_FORMAT_PARAM = "xml"; // NOI18N
     private static final String EXCLUDE_PARAM = "--exclude"; // NOI18N
     private static final String SUFFIXES_PARAM = "--suffixes"; // NOI18N
+    public static final String EMPTY_RULE_SET = "<none>"; // NOI18N
```

Comment:
I would consider `i18N` because it is not a command or parameter.

Reply:
It's kind of special rule - ignored during command parameters building, but still written into settings.
Making it translated would also mean to map value saved into settings to translated value shown in list, right?

Reply:
I agree with Junichi, this should be translated. However, the stored value should be just some identificator (so, not the translated string).

Reply:
Can you point me to a place in NetBeans where something like this is done or what class/interface to look for, please.

Reply:
Not sure what you mean... Do you mean a custom Renderer for the given JList so this specific option can be translated?

Reply:
Yes, that are the keywords I was looking for ;)
I'm novice in Java/NetBeans realm and I try to learn it by observing how these things are done in other parts of the project.

Reply:
This is more about Java and Swing than NetBeans. See [the official UI tutorial](https://docs.oracle.com/javase/tutorial/uiswing/components/list.html). This tutorial is fine IMHO and I am pretty sure that you will be able to find a similar case already used in NetBeans ;)

Reply:
I created renderer based on `PHPStanLevelListCellRenderer`.
Value is now only dash '-' and '<none>' is shown in list.
Before testing again you should clean userdir.